DEVELOPMENT.md

```diff
@@ -1,40 +1,80 @@
+
+If you are submitting a Pull Request (PR) to provide a fix or feature for the action, the following is relevant to you:
+
+The main source code elements for the action are:
+- [action.yml](action.yml)
+- [index.js](index.js)
+- [src/ping.js](src/ping.js)
+
+The action runs from the [dist](dist) directory, which requires a `build` step to set up. This step uses [@vercel/ncc](https://github.com/vercel/ncc) to compile the code and modules into one file used for distribution. See below:
+
+1. If you are a new external contributor, then first fork the repository (see GitHub documentation [About forks](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/about-forks)).
+1. Create a new branch with a meaningful name e.g. `fix/my-bug` based on the current `master` branch.
+1. Make the necessary source code changes, including additions or changes to the [README.md](./README.md) documentation if parameters are added or affected.
+1. Ensure you are using [Node.js LTS](https://nodejs.org/en).
+1. Execute the following in the root directory of the cloned repository
+
+```bash
+npm install
+npm run format
+npm run build
+```
+
+6. Commit the change. (If you are working on Microsoft Windows, see [Windows users](#windows-users) below.)
+1. Push to the repository.
+1. If you are working in a fork, ensure actions are enabled.
+1. Refer to [Manually running a workflow](https://docs.github.com/en/actions/managing-workflow-runs/manually-running-a-workflow) to use this to test any particular workflow affected by your change.
+1. Open a draft PR with a title which starts with `fix:` or `feat:` if a new release should be triggered by the PR. For more details on the release process, see the section [Merging pull requests](#merging-pull-requests) below.
+1. Note that first-time submitters must sign the CLA agreement and be approved to run workflows.
+1. Check that all tests which are triggered by the PR were either successful or were skipped.
+1. If the PR is showing that the tests were successful mark the PR as Ready for Review.
+
+### Windows users
+
+The repository is set up with a `git` / `Husky` pre-commit hook which ensures that any changes to the core source files are formatted and built consistently before they are committed. This does not work well with [GitHub Desktop](https://docs.github.com/en/desktop). You can disable this function by setting an environment variable `HUSKY=0`. If you do this and then omit to use `npm run format` / `npm run build` before you commit any changes affecting the core source files, you will find that checks fail when you submit a PR. In this case you should run the format and build commands and amend your commit.
+
+## Adding a new example
+
+1. If you are creating a new example, add this as a new project in the `examples` directory (and if appropriate also in the `examples/v9` directory). An example project is a regular NPM package with its own `package.json` and Cypress dev dependency.
```

Comment:
```suggestion
1. If you are creating a new example, add this as a new project in the `examples` directory. An example project is a regular NPM package with its own `package.json` and Cypress dev dependency.
```

At this point, we do not need to add additional examples / support to Cypress v9. At some point (doesn't need to be anytime soon) we will want to remove the major version delineation as our typical standard is to only provide improvements and fixes to the latest major version.

Reply:
@emilyrohrbough 

Thank you very much for that decision! I wasn't sure what the policy should be. That certainly makes it simpler from the documentation and maintenance side.